docs/reference/contactproperties.md

```diff
@@ -53,12 +54,21 @@ With three values, the first solid (corresponding to `material1`) uses asymmetri
 Finally, with four values, both solids use asymmetric coefficients, first two for the first solid and last two for the second solid.
 The two friction directions are defined for each faces of the geometric primitives and match with the U and V components used in the texture mapping.
 Only the `Box`, `Plane` and `Cylinder` primitives support asymmetric friction.
-If another primitive is used, only the first value will be used for symetric friction.
-WEBOTS\_HOME/projects/sample/howto/asymmetric\_friction/worlds/asymmetric\_friction1.wbt contains an example of fully asymmetric friction.
+If another primitive is used, only the first value will be used for symmetric friction.
+The world [asymmetric\_friction1.wbt](../guide/samples-howto.md#asymmetric_friction1-wbt) contains an example of fully asymmetric friction.
```

Comment:
We rather use this style although both styles are probably correct:
```suggestion
The [asymmetric\_friction1.wbt](../guide/samples-howto.md#asymmetric_friction1-wbt) world contains an example of fully asymmetric friction.
```
(see other occurrences below)

*Note*: "the readme.txt file" has 1.2M hits in google search while "the file readme.txt" has only 88k.